backup/queries_table_defs.go

```diff
@@ -302,10 +303,7 @@ func GetColumnDefinitions(connectionPool *dbconn.DBConn) map[uint32][]ColumnDefi
 		CASE WHEN a.attstorage != t.typstorage THEN a.attstorage ELSE '' END AS storagetype,
 		coalesce('('||pg_catalog.pg_get_expr(ad.adbin, ad.adrelid)||')', '') AS defaultval,
 		coalesce(d.description, '') AS comment,
-		CASE
-			WHEN a.attacl IS NULL THEN NULL
-			WHEN array_upper(a.attacl, 1) = 0 THEN a.attacl[0]
-			ELSE unnest(a.attacl) END AS privileges,
+		a.attacl AS privileges,
 		CASE
```

Comment:
I think, we may remove this and rewrite logic related to kind Empty

Reply:
Check other places with `Empty` word in code and test, for example
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl.go#L159
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl.go#L191
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl.go#L571
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl_test.go#L313
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl_test.go#L430
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/queries_acl.go#L168
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/queries_acl.go#L347
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/queries_table_defs.go#L311
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/queries_table_defs.go#L359
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/queries_acl_test.go#L103
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl_test.go#L358
https://github.com/arenadata/gpbackup/blob/e75a86aa7c785f35f6f24cdc903828b8b481bba8/backup/predata_acl_test.go#L474
and so on

Reply:
The other places (that are yet not changed) are related to other structs like `MetadataQueryStruct` , not to `ColumnDefinition`. So, changing other places will be refactoring, that we do not want to mix with the bug fix.